# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lua-random/lua-random-make-no-test.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Do not run 'test' make target by default

--- random/Makefile.install	2010-10-21 14:24:49.438696310 +0200
+++ random/Makefile	2010-10-21 14:25:19.606696291 +0200
@@ -22,9 +22,9 @@ T= $(MYNAME).so
 OBJS= $(MYLIB).o
 TEST= test.lua
 
-all:	test
+all:	$T
 
-test:	$T
+test:	all
 	$(LUABIN)/lua $(TEST)
 
 o:	$(MYLIB).o
